### ISSUE TITLE
[Accessibility] - Possibility to have ngbNavButton instead of ngbNavLink

### ISSUE THREAD

**Issue by GregoireLgt** [ng-bootstrap/ng-bootstrap#4204]
### For accessibility purposes, 

is it possible to have the ngbNavButton directive instead of ngbNavLink ? The ngbNavLink directive adds role="tab" to the link (`<a>` tag) in which you put this directive but according to the W3C, it is not recommended to have `<a>` with role="tab" attribute as you can see here : 

https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html

role="tab" requires a `<button>`

To sum up, 

Add the possibility to have `<button ngbNavButton>` instead of `<a ngbNavLink>`

**EDIT : corresponding PR here : https://github.com/ng-bootstrap/ng-bootstrap/pull/4206**

### Versions of Angular, ng-bootstrap:

Angular 13 and more
ng-bootstrap 11.0.0 and more

Thanks a lot

**Comment by maxokorokov** [comment]
I think we should fix it indeed.
Also see https://getbootstrap.com/docs/5.1/components/navs-tabs/#javascript-behavior

**Comment by bastienmoulia** [comment]
Can we add this for future milestones?
